src/main/kotlin/org/idea_sp/SourcePawnColorSettingsPage.kt

```diff
@@ -10,10 +11,30 @@ import javax.swing.Icon
 class SourcePawnColorSettingsPage : ColorSettingsPage {
     companion object {
         private val DESCRIPTORS = arrayOf(
-            AttributesDescriptor("Operator", SourcePawnSyntaxHighlighter.OPERATOR),
-            AttributesDescriptor("Literal", SourcePawnSyntaxHighlighter.LITERAL),
-            AttributesDescriptor("Reserved", SourcePawnSyntaxHighlighter.RESERVED),
-            AttributesDescriptor("Comment", SourcePawnSyntaxHighlighter.COMMENT),
+            AttributesDescriptor(
+                "Operator", TextAttributesKey.createTextAttributesKey(
+                    "SOURCE_PAWN_OPERATOR",
+                    DefaultLanguageHighlighterColors.OPERATION_SIGN
+                )
+            ),
+            AttributesDescriptor(
+                "Literal", TextAttributesKey.createTextAttributesKey(
+                    "SOURCE_PAWN_CONSTANT",
+                    DefaultLanguageHighlighterColors.CONSTANT
+                )
+            ),
+            AttributesDescriptor(
+                "Reserved", TextAttributesKey.createTextAttributesKey(
+                    "SOURCE_PAWN_RESERVED",
+                    DefaultLanguageHighlighterColors.KEYWORD
+                )
+            ),
+            AttributesDescriptor(
+                "Comment", TextAttributesKey.createTextAttributesKey(
+                    "SOURCE_PAWN_COMMENT",
+                    DefaultLanguageHighlighterColors.LINE_COMMENT
+                )
+            ),
```

Comment:
Whoops, this was a mistake — I meant to inline some variables in another file.

Also, this settings page will need some changing after the syntax-highlighter is all settled.